std/process.d

```diff
@@ -1443,6 +1499,33 @@ int wait(Pid pid) @safe
     else version (Posix) assert(pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "do { timeout 1 } while (1)";
```

Comment:
> Try inserting Thread.sleep(1.seconds) before the kill. I'm guessing your test is killing the process before the OS figures out the test script is invalid, in which case it's at least a race condition.

Great observation! I totally missed that this could happen.